assets/capability/capabilities/measure_power.json

```diff
@@ -16,15 +16,15 @@
   },
   "insights": true,
   "desc": {
-    "en": "Power in Watt (W)",
+    "en": "Power in watt (W)",
     "nl": "Vermogen in Watt (W)",
```

Comment:
Watt mag in het Nederlands ook met een kleine letter geschreven worden volgens mij

Reply:
Fixed